recipes/turbo_turtle/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "turbo_turtle" %}
+{% set version = "0.12.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lanl-aea/turbo-turtle/archive/refs/tags/{{ version }}.tar.gz
+  sha256: e89f218c8d95484f3b0b8b61427c0bd82779226301436c991535966bac4d140d
+
+build:
+  noarch: python
+  number: 0
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  entry_points:
+    - turbo-turtle = turbo_turtle._main:main
+  ignore_run_exports:
+    - python_abi
+
+requirements:
+  build:
```

Comment:
These should all be in host

Reply:
I'm using the build/host separation for PREFIX/BUILD_PREFIX separation and trying to follow the build/host separation in the conda-build documentation: https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#requirements

Am I interpreting the build/host separation incorrectly for a noarch package? Does conda-forge have documented preferences for build/host section use for pure Python packages?

Reply:
Generally, Python packages should be in a host as they are specific to the target. In the `noarch` case, `build` should be empty`